Library/Config.ts

```diff
@@ -46,28 +45,26 @@ class Config {
             "*.blob.core.cloudapi.de",
             "*.blob.core.usgovcloudapi.net"];
         
-        this.correlationId = CorrelationIdManager.correlationIdPrefix; // Initialize with a blank correlation ID until we fetch the correct value.
-
         this.setCorrelationId = (correlationId) => this.correlationId = correlationId;
 
-        CorrelationIdManager.queryCorrelationId(
-            this.profileQueryEndpoint,
-            this.instrumentationKey,
-            this.correlationIdRetryIntervalMs,
-            this.setCorrelationId);
+        this.profileQueryEndpoint = process.env[Config.ENV_profileQueryEndpoint] || this.endpointBase;
```

Comment:
Should this be setting the private property directly instead? Or does the public one delay the appid query as well?